<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WSoundBotMainUiClass</class>
 <widget class="QWidget" name="WSoundBotMainUiClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WSoundBotMainUi</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	font: 12px &quot;微软雅黑&quot;;
}

QFrame#frame_left{
	border:1px;
	border-right:1px solid rgba(80,80,80,125);
}

QPushButton{
	border:none;
	background-position: center;
    background-repeat: no-repeat;
}

QPushButton#pb_back{
	background-image: url(:/QtTest/icon/Setting/soundbot/back.png);
}

QCheckBox#checkBox::indicator {width: 30px; height: 30px;}

QCheckBox#checkBox::indicator:unchecked {image: url(:/QtTest/icon/Setting/off.png);}

QCheckBox#checkBox::indicator:checked {image: url(:/QtTest/icon/Setting/open.png);}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>64</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>64</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pb_back">
        <property name="minimumSize">
         <size>
          <width>22</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>22</width>
          <height>28</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_back_1">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>28</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	font: 24px &quot;微软雅黑&quot;;
	color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 #0066FF, stop:1 #BD00FF);
}</string>
        </property>
        <property name="text">
         <string>Soundbot</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>517</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string>Activated</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="frame_left">
       <property name="minimumSize">
        <size>
         <width>190</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>190</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="WNavbarButton" name="pb_sound_page1">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="WNavbarButton" name="pb_sound_page2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="WNavbarButton" name="pb_sound_page3">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="WNavbarButton" name="pb_sound_page4">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="WNavbarButton" name="pb_sound_page5">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="WNavbarButton" name="pb_sound_page6">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>216</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <widget class="WSoundSourcePage" name="source_page"/>
       <widget class="WTranslationPage" name="trans_page"/>
       <widget class="WVoiceLibPage" name="voice_lib_page"/>
       <widget class="WMyVoicePage" name="myVoice_page"/>
       <widget class="WSpeechSysPage" name="speechSys_page"/>
       <widget class="WHistoryDiaPage" name="history_dia_page"/>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>WNavbarButton</class>
   <extends>QPushButton</extends>
   <header>widget/model/WNavbarButton.h</header>
  </customwidget>
  <customwidget>
   <class>WSoundSourcePage</class>
   <extends>QWidget</extends>
   <header>WSoundSourcePage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WTranslationPage</class>
   <extends>QWidget</extends>
   <header>WTranslationPage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WVoiceLibPage</class>
   <extends>QWidget</extends>
   <header>WVoiceLibPage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WMyVoicePage</class>
   <extends>QWidget</extends>
   <header>WMyVoicePage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WSpeechSysPage</class>
   <extends>QWidget</extends>
   <header>WSpeechSysPage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WHistoryDiaPage</class>
   <extends>QWidget</extends>
   <header>WHistoryDiaPage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../AiSound.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pb_back</sender>
   <signal>clicked()</signal>
   <receiver>WSoundBotMainUiClass</receiver>
   <slot>slot_sound_back()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>24</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>105</x>
     <y>-11</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_back_1</sender>
   <signal>clicked()</signal>
   <receiver>WSoundBotMainUiClass</receiver>
   <slot>slot_sound_back()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>66</x>
     <y>-15</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_sound_back()</slot>
 </slots>
</ui>
